src/Carousel.vue

```diff
@@ -406,9 +406,10 @@ export default {
      * @return {Number}
      */
     maxOffset() {
-      return (
+      return Math.max(
```

Comment:
I see that this change for your example works and also doesn't break the other functionality. The only issue I see is that the pagination breaks but that seems to just be an issue with scrollPerPage.

Reply:
Yeah @ashleysimpson and I discussed, this seems like an issue with the `scrollPerPage` api. We'll open an issue to address